ghist/cogs/spelunkicon.py

```diff
@@ -64,13 +72,17 @@ async def spelunkicon(self, ctx, *, orig_words=None):
         if random_size:
             generator = random.Random(word)
             size = generator.choice(range(3, 9))
+
+        if size is not None:
             url += f"&size={size}"
-        elif big:
-            url += "&size=8"
-        elif small:
-            url += "&size=4"
 
         if pride:
             url += "&egg=pride"
 
+        if classic:
+            url += "&egg=classic"
+
+        if chaos:
```

Comment:
Doesn't look like you configured this one. no variable was ever initialized or set from options

Reply:
🧠